network/api/apiFunctions.js

```diff
@@ -12,22 +12,28 @@ export async function addNode(partition, nodeID, data, tableName) {
         Item: {
             Partition: partition,
             NodeID: nodeID,
-            Data: data, // data will be expanded to more keys and values later
+            Name: data.name,
+            Phone: data.phone, 
+            Status: data.status, 
+            Request: data.request, 
+            Reminder: data.reminder, 
+            CustomFields: data.customFields,
```

Comment:
this needs to be "customFields" (lowercase "C"). This should be changed in the future, but it will be much easier right now to just do it here.